performance_regression_test.py

```diff
@@ -277,6 +277,75 @@ def test_uniform_counter_update_bench(self):
         self.display_results(results, test_name='test_read_bench')
         self.check_regression()
 
+    def test_mv(self):
+        """
+        Test steps:
+
+        1. Run WRITE workload without materialized view
+        2. Run WRITE workload with materialized view one or more times
```

Comment:
Please update to the exact times we run it according to the current yaml and the deifference bewteen them.
For example:
2. Run WRITE workload with MV when View is on .... (clustering key is the same host as partition key).
3. Drop View.
4. Run ...